dockerfiles/testsuite/README.md

```diff
@@ -8,9 +8,9 @@ docker-compose up -d
 go run ../../cmd/vochaintest/vochaintest.go --oracleKey $(. env.oracle0key; echo $DVOTE_ETHCONFIG_SIGNINGKEY) --electionSize=1000
 ```
 
-there's also a bash script:
+there's also a bash script, which prefers to be run with `NOTTY=1`
```

Comment:
Why this?

Reply:
tests fail
```
[+] Running 2/0
 ⠿ Container testsuite-seed-1      Running                                                          0.0s
 ⠿ Container testsuite-gateway0-1  Running                                                          0.0s
[+] Running 1/1
 ⠿ Container testsuite-test-1  Started                                                              0.3s
the input device is not a TTY
### Running test merkle_vote_plaintext concurrently with others ###
[+] Running 2/0
 ⠿ Container testsuite-seed-1      Running                                                          0.0s
 ⠿ Container testsuite-gateway0-1  Running                                                          0.0s
[+] Running 1/1
 ⠿ Container testsuite-test-1  Started                                                              0.3s
the input device is not a TTY
### Running test cspvoting concurrently with others ###
[+] Running 2/0
 ⠿ Container testsuite-seed-1      Running                                                          0.0s
 ⠿ Container testsuite-gateway0-1  Running                                                          0.0s
[+] Running 1/1
 ⠿ Container testsuite-test-1  Started                                                              0.3s
the input device is not a TTY
### Running test anonvoting concurrently with others ###
[+] Running 2/0
 ⠿ Container testsuite-seed-1      Running                                                          0.0s
 ⠿ Container testsuite-gateway0-1  Running                                                          0.0s
[+] Running 1/1
 ⠿ Container testsuite-test-1  Started                                                              0.3s
the input device is not a TTY
### Running test merkle_vote_encrypted concurrently with others ###
[+] Running 2/0
 ⠿ Container testsuite-seed-1      Running                                                          0.0s
 ⠿ Container testsuite-gateway0-1  Running                                                          0.0s
[+] Running 1/1
 ⠿ Container testsuite-test-1  Started                                                              0.4s
the input device is not a TTY
### Waiting for tests to finish ###
Vochain test vocli finished correctly!
Vochain test tokentransactions failed!
```

Reply:
I think is because you're using `zsh`. If you use bash directly there is no problem, btw I ran into the same problem :') 
I would rather advocate for using bash directly and aware that if there is some TTY problem there is two ways of solving it:
1. Change to bash by typing bash in the console 
2. Run with `NOTTY=1`

Reply:
What are your thoughts on this ?

Reply:
I'm also using bash.